### ISSUE TITLE
Increasing Access to p5.js – Fix Broken Link on Ace Editor Learn Page

### ISSUE THREAD

**Issue by Mamatha1718** [processing/p5.js#7590]
Description:
Hello, I was trying to learn about embedded statements using Ace Editor, but the "Learn page" link is not working. It gives a 404 File Not Found error. I have attached a screenshot of the error and the documentation code.
References: https://github.com/processing/p5.js/wiki/Embedding-p5.js

![Image](https://github.com/user-attachments/assets/c02d71ac-e972-4b45-b9bb-5bd1c0d11f45)

![Image](https://github.com/user-attachments/assets/d27f496f-6989-4055-a168-5d3217014e62)

How would this new feature help increase access to p5.js?
Fixing this broken link will help users access important learning resources on embedding p5.js sketches, improving their ability to use p5.js effectively.
Steps to Reproduce:
1. Open the wiki/Embedding-p5.js Page.
2. Click on the "Learn page" link in Ace Editor section.

Thank You.

### Most appropriate sub-area of p5.js?

- [x] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### Feature enhancement details

Hi Everyone, I would like to understand all embedded methods, in Embedded-p5.js file Ace Editor section learn page link give 404 file not found error,
Issue:

The link is not accessible, preventing users from learning about Ace Editor integration with p5.js.
Expected Outcome:

The link should direct users to the correct tutorial or documentation page.
Request:
Can someone look into this and update the link to a working resource?
Thank You.

**Comment by limzykenneth** [comment]
I've fixed the link by linking it to the [archive version of the site](http://archive.p5js.org/learn) however we are not keeping the wiki up to date at the moment as we aim to consolidate contents on the website itself. The new website also no longer uses Ace to I linked to the archive site instead.

**Comment by Mamatha1718** [comment]
@limzykenneth ,Thank You for solving this, and Thank You for your clarification.
